doc/man1/info.pod

```diff
@@ -0,0 +1,76 @@
+=pod
+
+=head1 NAME
+
+openssl-info,
+info - print OpenSSL built-in information
+
+=head1 SYNOPSIS
+
+B<openssl info>
+[B<-help>]
+[B<-configdir> | B<-c>]
+[B<-enginesdir> | B<-e>]
+[B<-modulesdir> | B<-m>]
+[B<-dsoext>]
+[B<-dirfilesep>]
+[B<-listsep]>
+
+=head1 DESCRIPTION
+
+This command is used to print out info information about OpenSSL.
```

Comment:
info information?

Reply:
Yes, exactly.  It's info about the information that you will eventually...

...  nah, kidding, will fix.

Reply:
I ended up adding a bit more